tests/tests_geomstats/test_discrete_curves.py

```diff
@@ -550,3 +549,24 @@ def test_horizontal_geodesic(self):
         result = gs.sum(vertical_norms ** 2, axis=1) ** (1 / 2)
         expected = gs.zeros(n_times - 1)
         self.assertAllClose(result, expected, atol=1e-3)
+
+    @geomstats.tests.np_autograd_and_torch_only
+    def test_quotient_dist(self):
+        """Test quotient distance.
+
+        Check that the quotient distance is the same as the distance
+        between the end points of the horizontal geodesic.
+        """
+        curve_a_resampled = self.curve_fun_a(self.sampling_times ** 2)
+        curve_b = gs.transpose(
+            gs.stack(
+                (
+                    gs.zeros(self.n_sampling_points),
+                    gs.zeros(self.n_sampling_points),
+                    gs.linspace(1.0, 0.5, self.n_sampling_points),
+                )
+            )
+        )
+        result = self.quotient_srv_metric_r3.quotient_dist(curve_a_resampled, curve_b)
+        expected = self.quotient_srv_metric_r3.quotient_dist(self.curve_a, curve_b)
```

Comment:
Why not calling `quotient_dist` just `dist`? This variable name seems redundant: "quotient_metric.quotient_dist".